### DISCUSSION TITLE
How I can generate images using a base template

### DISCUSSION THREAD

**Discussion by Hnd-jg** [comfyanonymous/ComfyUI#1255] [Q&A]
Hi guys, I´ve been trying to generate styles on this template
![TEMPLATE](https://github.com/comfyanonymous/ComfyUI/assets/142596933/f85c1626-b092-4c52-baf6-ebd8ed2b6712)
But I don´t get the results that I want.
![image](https://github.com/comfyanonymous/ComfyUI/assets/142596933/4234b8cc-f841-4528-830b-3916e3dff733)
I get stuff like the last image, but I want to get images like this:
![WhatsApp Image 2023-08-16 at 11 56 37 PM](https://github.com/comfyanonymous/ComfyUI/assets/142596933/ebd265b3-ff52-4db9-bead-6d61b70b1269)
![18](https://github.com/comfyanonymous/ComfyUI/assets/142596933/067d18c4-8ce9-45ac-95be-b2fd349f2429)

What I can do?

**Comment by williamcboehmjr** [comment]
Have you tried controlnet where you could turn the denoise almost all the way to 1.000 and make sure the control net is at max. Canny and depth would do well with this. I would also use the refiner a tad. Use exponential scheduler on a refiner 2nd stage, I've been seeing it fixes or it does things a bit more structured. They even have Lora's for buildings to make sure details for buildings are more or less depending on the strength of the Lora.

**Comment by EpproxZimlon** [comment]
Just want to say that when people post images of completed workflows like you did above, I learn this program far faster than any YouTube video. So thank you...
![image](https://github.com/comfyanonymous/ComfyUI/assets/37718128/cd0948b2-1d18-478f-96d0-2ce51213e0f9)
